docs/source/trustme-trio-example.py

```diff
@@ -1,11 +1,15 @@
 # trustme-trio-example.py
 
+import ssl
+
 import trustme
 import trio
 
 # Create our fake certificates
 ca = trustme.CA()
+client_ca = trustme.CA()
 server_cert = ca.issue_cert(u"test-host.example.org")
+client_cert = client_ca.issue_cert(u"@webknjaz here")
```

Comment:
trustme's heuristic is that any id that contains an `@` sign should be tagged as an email address, so let's use a valid email address :-). Maybe `u"client@example.org"`?

Reply:
I'd prefer `common_name=` once #37 is merged

Reply:
Either CN or a SAN are valid here. I guess most real client cert deployments either use SANs or use the whole Distinguished Name (which the CN is one field in, but not necessarily the most important).

I think all else being equal we should nudge people towards using SANs. And we don't provide any way to control the Distinguished Name as a whole. And adding extra kwargs complicates the example. So let's use `"client@example.org"`.